### ISSUE TITLE
exclude empty setup/teardown from --durations

### ISSUE THREAD

**Issue by azrdev** [pytest-dev/pytest#4063]
Write a test without setup or tear-down:
~~~python
def test_sleep():
    import time
    time.sleep(2)
~~~

Run `pytest --durations 0` on this test. Observe something like 
~~~
======================= slowest test durations =======================
2.00s call     tests/test.py::test_sleep
0.00s teardown tests/test.py::test_sleep
0.00s setup    tests/test.py::test_sleep
~~~

On medium-size testsuites, where one would like to see timings off all tests that run, these useless 0.00s entries clutter the output, so I'd suggest to remove them.

# Versions
python-pytest 3.8.1-1
python 3.7.0-4
from archlinux repos

**Comment by nicoddemus** [comment]
Sounds good. I believe your suggestion should be the default, but we fallback to show everything if the user passes `-vv`. 👍

**Comment by pecey** [comment]
So the current behaviour should happen if the user passes `-vv`, else we should only report values which are non-zero?

**Comment by azrdev** [comment]
@pecey  IMHO that would be misleading. We should only report values that were actually run, i.e. exclude setup/teardown exactly if they were not specified by the user.

**Comment by crazymerlyn** [comment]
Fixed by #4086.
